### ISSUE TITLE
Question about the setting of weight decay.

### ISSUE THREAD

**Issue by YifanXu74** [THUDM/CogVLM#101]
Hi, very nice work!

I have noticed that CogVLM is trained with different weight decay values in each stage. For example, 0.05 for pre-training and 0.1 for ﬁnetuning. Why doing this difference and what is the strategy to set these values?

**Comment by Sleepychord** [comment]
Oh, we actually did not carefully investigate it. It is more because different authors are using their own defaults.
